content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,264 @@
+single instance of Cluster Autoscaler can simultaneously manage multiple
+Node groups. When provisioning, Cluster Autoscaler will add Nodes to the group
+that best fits the requests of pending Pods. When consolidating, Cluster Autoscaler
+always selects specific Nodes to remove as opposed to just resizing the underlying VM group.
+
+Main differences vs Karpenter:
+* Cluster Autoscaler (CA) provides features related to just Node autoscaling. Node lifecycle
+  beyond that is not in its scope.
+* CA doesn't support auto-provisioning, the Node groups it can provision from have to be
+  pre-configured.
+
+Additional context:
+* Documentation overview: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md
+* Supported cloud providers: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md#faqdocumentation
+* Extensive FAQ: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md
+* Contact: https://github.com/kubernetes/community/tree/master/sig-autoscaling#contact
+
+#### Karpenter
+
+Karpenter auto-provisions VMs based on [NodePool](https://karpenter.sh/docs/concepts/nodepools/)
+configuration provided by the user. Karpenter handles all aspects of node lifecycle, not just autoscaling.
+It works directly with VMs and doesn't rely on provider-side VM groups.
+
+Main differences vs Cluster Autoscaler (CA):
+* In addition to Node autoscaling, Karpenter provides features intended for managing
+  Node lifecycle altogether (for example grouping Nodes into pools, handling Node upgrades).
+* Karpenter supports auto-provisioning, the user only has to configure a set of
+  constraints for the provisioned Nodes instead of pre-configuring Node groups explicitly.
+
+Additional context:
+* Documentation: https://karpenter.sh/
+* Supported cloud providers: TBD
+* Extensive FAQ: https://karpenter.sh/docs/faq/
+* Contact: https://github.com/kubernetes-sigs/karpenter#community-discussion-contribution-and-support
+
+## Configure workloads for autoscaling {#workload-config}
+
+Normally, workloads don't have to be additionally configured to work with Node
+autoscaling, beyond just configuring the autoscaler. However, autoscalers provide
+the ability to change their behavior based on workload characteristics. The
+configuration options documented here are supported by all official autoscalers.
+Thanks to this, the workloads can be portable between clusters using different autoscalers.
+Additionally, the autoscalers provide their own, non-portable configuration
+options (see autoscaler-specific documentation).
+
+### Configure a workload not to be disrupted by consolidation
+{{< feature-state state="stable" for_k8s_version="v1.31" >}}
+
+You can use annotation `autoscaling.kubernetes.io/do-not-disrupt: true` on a Pod to
+prevent it being disrupted by consolidation. Consolidation should never remove a Node
+where a Pod annotated with `autoscaling.kubernetes.io/do-not-disrupt: true` is running.
```

Comment:
Is this aspirational? Or is this the current CAS annotation that's respected? This is definitely not the annotation that we use in Karpenter today. Maybe we should include the `karpenter.sh/do-not-disrupt` annotation in here until we align on those if CAS has already made the migration?